<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateUser</class>
 <widget class="QWidget" name="CreateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="validerPushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>94</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Confirmation (mot de passe)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenomLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="motDePasseLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Mot de passe</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="adminCheckBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Prénom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>68</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Identifiant</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="identifiantLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>33</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>377</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Création d'un utilisateur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmationMotDePasseLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
